doc/visual-programming/source/widgets/visualize/scatterplot.md

```diff
@@ -45,6 +45,10 @@ Intelligent Data Visualization
 
 If a dataset has many attributes, it is impossible to manually scan through all the pairs to find interesting or useful scatter plots. Orange implements intelligent data visualization with the **Find Informative Projections** option in the widget. The goal of optimization is to find scatter plot projections where instances are well separated.
 
+If categorical variable is selected in the Color section, the option will score variable combinations based on how well class values are separated. For each data instance, the scoring will check 10 closest neighbors and give a higher score to those combinations, where closest neighbor are of the same value.
```

Comment:
It is not clear that distances are measured in the projected space.
I don't like "of the same value". Maybe "have the same value"? The text also refers to "class value", although the color is not necessarily class. Also, "separation" is repeated from the last paragraph and thus doesn't add any new information. Maybe:

*If a categorical variable is selected in the Color section, the score is computed as follows. For each data instance, VizRank finds 10 nearest neighbours in the projected space, that is, on the figure as seen by the user. It then checks how many of them have the same color. The total VizRank score of the projection is then the average number of same-colored neighbours.*

*Computation for continuous colors is similar, except that the [coefficient of determination](https://en.wikipedia.org/wiki/Coefficient_of_determination) is used for measuring the local homogeneity of the projection.*

Reply:
Or, more compact:

*For each data instance, VizRank finds 10 nearest neighbours in the projected space, that is, on the figure as seen by the user. If a categorical variable is selected for coloring data points, VizRank checks how many neighbours have the same color. The total VizRank score of the projection is then the average number of same-colored neighbours. Computation for continuous colors is similar, except that the [coefficient of determination](https://en.wikipedia.org/wiki/Coefficient_of_determination) is used for measuring the local homogeneity of the projection.*